ibis/expr/types/relations.py

```diff
@@ -2012,6 +2012,21 @@ def filter(
         ]
         return an.apply_filter(self.op(), predicates).to_expr()
 
+    def nunique(self, where: ir.BooleanValue | None = None) -> ir.IntegerScalar:
+        """Compute the number of unique rows in the table.
+
+        Parameters
+        ----------
+        where
+            Optional boolean expression to filter rows when counting.
+
+        Returns
+        -------
+        IntegerScalar
+            Number of unique rows in the table
+        """
```

Comment:
Could you add a small example here as well (since all our other `Table` methods have examples)?

Reply:
Done!